src/ser.rs

```diff
@@ -466,7 +466,21 @@ impl<'a> Serializer<'a> {
         } else {
             drop(write!(self.dst, "\""));
         }
+        //let mut quotes: u8 = 0;
```

Comment:
I wrote this code because it was how I was expecting to solve this... it didn't work and instead I got the problem I posted about